src/helpers/getTrackers.js

```diff
@@ -0,0 +1,29 @@
+const { Contract, providers: { JsonRpcProvider } } = require('ethers')
+
+const getTrackers = async (address, config, jsonRpcProvider) => {
+    // eslint-disable-next-line import/no-dynamic-require,global-require
+    const trackerRegistryConfig = require(`../../configs/${config}`)
```

Comment:
You can do the following to avoid having to disable eslint
```js
const fs = require('fs')
...
JSON.parse(fs.readFileSync(`../../configs/${config}`))
```